lib/hanami/middleware/content_security_policy_nonce.rb

```diff
@@ -38,7 +38,7 @@ def call(env)
       private
 
       def random_nonce
-        SecureRandom.alphanumeric(8)
+        SecureRandom.urlsafe_base64(16)
```

Comment:
@timriley 👍 I peeked at another source than the Rails guides for inspiration – and an entrophy of 3.5 trillion seemed more than enough for an nonce. But since MDN also suggests 16bit, better safe than sorry. 😄